freeipa/src/main/java/com/sequenceiq/freeipa/service/upgrade/UpgradeService.java

```diff
@@ -130,4 +138,22 @@ private List<ImageInfoResponse> getTargetImages(String catalog, Stack stack, Ima
         LOGGER.debug("Found target images: {}", targetImages);
         return targetImages;
     }
+
+    String calculateUpgradeVariant(Stack stack, String accountId) {
```

Comment:
make this private please

Reply:
package protected to create UTs

Reply:
UT should test only public methods, and you shouldn't change visibility for tests. Either test via public method or refactor and move the method to separate class, because it means the class is already doing too much, so you are not able to unit test it